ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift

```diff
@@ -98,6 +98,9 @@ class SettingsViewController: TableViewController {
     self.cryptoStore = cryptoStore
     self.ipfsAPI = braveCore.ipfsAPI
 
+    self.keyringStore?.setupObservers()
+    self.cryptoStore?.setupObservers()
```

Comment:
Are these needed at initialization? We already have the observers setup in the `setUpSections` function below when Web3 row is selected (so observers are only setup once web3 settings view is opened, rather than when general app Settings is opened): https://github.com/brave/brave-core/blob/5425e704946fa88c86ffa789d315b468fec22edc/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift#L1344-L1345

Reply:
updated here [8193cb9](https://github.com/brave/brave-core/pull/22971/commits/8193cb9c02fb3fbabeac81084bb564414ca2c7e6)